iceoryx_hoofs/source/posix_wrapper/shared_memory_object/allocator.cpp

```diff
@@ -59,6 +64,8 @@ void* Allocator::allocate(const uint64_t size, const uint64_t alignment) noexcep
                   << " when there are already " << alignedPosition << " aligned bytes in use." << std::endl;
         std::cerr << "Only " << m_length - alignedPosition << " bytes left." << std::endl;
 
+        /// @todo replace Expects with IOX_ASSERT
+        // NOLINTNEXTLINE(hicpp-no-array-decay, cppcoreguidelines-pro-bounds-array-to-pointer-decay)
```

Comment:
With the recently added suppressions in `requires.hpp`, these warnings should not be present anymore. Can you check please?

Reply:
Redundant. Removed.